pom.xml

```diff
@@ -1028,6 +1028,54 @@
             </executions>
           </plugin>
           <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-dependency-plugin</artifactId>
+              <version>3.0.0</version>
+              <executions>
+                  <!-- TODO how to avoid downloading them all? -->
+                  <execution>
+                      <id>download node for Linux</id>
+                      <phase>initialize</phase>
+                      <goals>
+                          <goal>get</goal>
+                      </goals>
+                      <configuration>
+                          <artifact>com.github.eirslett:node:${node.version}:tar.gz:linux-x64</artifact>
```

Comment:
Yes I need to hardcode supported platforms and architectures. But this is no big deal as I anyway needed to manually upload all these artifacts [here](http://repo.jenkins-ci.org/public/com/github/eirslett/), so anyone needing to add a new version / platform / architecture will need to first

    mvn deploy:deploy-file -Dfile=node-v…-…-….tar.gz -DgroupId=com.github.eirslett -DartifactId=node -DgeneratePom=false -Dversion=… -Dpackaging=tar.gz -Dclassifier=…-… -Durl=https://repo.jenkins-ci.org/releases/ -DrepositoryId=maven.jenkins-ci.org

and then patch this POM to download it.